src/routers/user.ts

```diff
@@ -163,6 +172,54 @@ export const userRouter = () => {
     csrfProtectionMiddleware,
     get2faSignInController,
   );
+
+  userRouter.get(
+    "/double-authentication-choice",
+    checkUserIsConnectedMiddleware,
+    csrfProtectionMiddleware,
+    getDoubleAuthenticationChoiceController,
+  );
+
+  userRouter.get(
+    "/configuring-single-use-code",
+    checkUserIsConnectedMiddleware,
+    csrfProtectionMiddleware,
+    getConfiguringSingleUseCodeController,
+  );
+
+  userRouter.get(
+    "/authenticator-app-configuration",
+    checkUserIsConnectedMiddleware,
+    csrfProtectionMiddleware,
+    getAuthenticatorAppConfigurationController,
+  );
+
+  userRouter.post(
+    "/authenticator-app-configuration",
+    nocache(),
+    urlencoded({ extended: false }),
```

Comment:
todo: remove this redoundant middlewares

Reply:
why it is redoundant ?